first_year_scraper/excel_formatter.py

```diff
@@ -0,0 +1,137 @@
+from openpyxl import Workbook
+from openpyxl import load_workbook
```

Comment:
Since there are a lot of imports add a requirements.txt maybe with the versions they are working on.

Reply:
Thanks for the suggestion! I've pushed the changes